interfaces/apparmor/template_vars.go

```diff
@@ -47,15 +47,15 @@ func legacyVariables(appInfo *snap.AppInfo) []byte {
 	fmt.Fprintf(&buf, "@{APP_APPNAME}=\"%s\"\n", appInfo.Name)
 	// TODO: replace with app.SecurityTag()
 	fmt.Fprintf(&buf, "@{APP_ID_DBUS}=\"%s\"\n",
-		dbus.SafePath(fmt.Sprintf("%s.%s_%s_%s",
-			appInfo.Snap.Name(), appInfo.Snap.Developer, appInfo.Name, appInfo.Snap.Version)))
+		dbus.SafePath(fmt.Sprintf("%s.%s_%s_%d",
+			appInfo.Snap.Name(), appInfo.Snap.Developer, appInfo.Name, appInfo.Snap.Revision)))
 	// XXX: How is this different from APP_ID_DBUS?
 	fmt.Fprintf(&buf, "@{APP_PKGNAME_DBUS}=\"%s\"\n",
 		dbus.SafePath(fmt.Sprintf("%s.%s", appInfo.Snap.Name(), appInfo.Snap.Developer)))
 	// TODO: stop using .Developer, investigate how this is used.
 	fmt.Fprintf(&buf, "@{APP_PKGNAME}=\"%s\"\n", appInfo.Snap.Name())
 	// TODO: switch to .Revision
-	fmt.Fprintf(&buf, "@{APP_VERSION}=\"%s\"\n", appInfo.Snap.Version)
+	fmt.Fprintf(&buf, "@{APP_VERSION}=\"%d\"\n", appInfo.Snap.Revision)
```

Comment:
We want ``SNAP_REVISION`` but I'd like to do that later (next week) as this is internal to apparmor.